src/components/App.tsx

```diff
@@ -6037,8 +6038,6 @@ class App extends React.Component<AppProps, AppState> {
               event[KEYS.CTRL_OR_CMD] ? null : this.state.gridSize,
             );
 
-          this.maybeSuggestBindingForAll(selectedElements);
```

Comment:
Removing this so a bindable element isn't highlighted the arrow is moved as a whole, and one of its endpoints is within the binding gap the element.